target/linux/ramips/image/mt76x8.mk

```diff
@@ -17,6 +18,11 @@ define Build/sercom-footer
 	$(call Build/sercom-seal,-f)
 endef
 
+define Build/jcg-header
```

Comment:
Can you please unify this helper command definition with the one in rt305x.mk's Build/jcg-header and move it to ramips/image/Makefile?

(Please also move the DEVICE_VARS += JCG_MAXSIZE there).